### ISSUE TITLE
Unable to replicate gnn result

### ISSUE THREAD

**Issue by nanjiangwill** [thunlp/FewRel#61]
Hi, I was unable to replicate gnn + cnn result on any n way k shot experiment, and the accuracy was always 1/N, what might be the problem.

Sometimes, the experiment goes well when I change the seed

**Comment by gaotianyu1350** [comment]
Hi,

We also found that GNN training could be quite unstable sometimes. Making sure you use the latest pytorch version, changing random seeds or trying larger batch sizes may help.
